apps/token/src/routes/rewards/epoch-individual-rewards/epoch-individual-rewards-table.tsx

```diff
@@ -0,0 +1,100 @@
+import { formatNumber, toBigNum } from '@vegaprotocol/react-helpers';
+import { Tooltip } from '@vegaprotocol/ui-toolkit';
+import { useAppState } from '../../../contexts/app-state/app-state-context';
+import {
+  rowGridItemStyles,
+  RewardsTable,
+} from '../shared-rewards-table-assets/shared-rewards-table-assets';
+import type { EpochIndividualReward } from './generate-epoch-individual-rewards-list';
+
+interface EpochIndividualRewardsGridProps {
+  data: EpochIndividualReward;
+}
+
+interface RewardItemProps {
+  value: string;
+  percentageOfTotal?: string;
+  dataTestId: string;
+  last?: boolean;
+}
+
+const DisplayReward = (reward: string, percentageOfTotal?: string) => {
+  const {
+    appState: { decimals },
+  } = useAppState();
+
+  if (Number(reward) === 0) {
+    return <span className="text-vega-dark-300">-</span>;
+  }
+
+  return (
+    <Tooltip
+      description={
+        <div className="flex flex-col items-start">
+          <span>{formatNumber(toBigNum(reward, decimals), decimals)}</span>
+          {percentageOfTotal && (
+            <span className="text-vega-dark-300">
+              ({percentageOfTotal}% of total distributed)
```

Comment:
question (blocking): i think this needs to be a translations string, right?

Reply:
Good spot, yes it does. Thank you!